release_notes/367-release-notes.adoc

```diff
@@ -0,0 +1,99 @@
+[id="367-release-notes"]
+= Red Hat Advanced Cluster Security for Kubernetes 3.67
```

Comment:
Can part of this be swapped for the {product-title} attribute? (without a preview I'm not sure what {product-title} renders as in this publication)

Reply:
I think because this is the title we have to write it fully. And after that we are importing common modules, so we can't use them in Titles.

Reply:
Aaah ok that is good to know - I knew I had seen it in some titles, but did not understand the workings behind that. Thank you!